kubernetes/keycloak/values.yaml

```diff
@@ -224,18 +224,18 @@ keycloak:
     deployPostgres: false
 
     # The database vendor. Can be either "postgres", "mysql", "mariadb", or "h2"
-    dbVendor: h2
+    dbVendor: postgres
 
     ## The following values only apply if "deployPostgres" is set to "false"
 
     # Specifies an existing secret to be used for the database password
-    existingSecret: ""
+    existingSecret: "keycloak-credentials"
 
     # The key in the existing secret that stores the password
     existingSecretKey: password
 
-    dbName: keycloak
-    dbHost: mykeycloak
+    dbName: keycloak_v2
+    dbHost: ""
```

Comment:
Is there a reason to pass this in at install rather than having it pull from a secret in k8?